src/ceph.in

```diff
@@ -552,8 +552,8 @@ def ping_monitor(cluster_handle, name, timeout):
         return 1
 
     mon_id = name[len('mon.'):]
+    run_in_thread(cluster_handle.connect, timeout=timeout)
```

Comment:
got it. if the monmap is only available from the monitor quorum, then we have no choice but to connect to the cluster for it before looking up the "mira021" in the monmap.

but i think the nice thing of "ceph ping mon.a" is that, it is a very thing wrapper around the `rados_ping_monitor()`, so one can ping a certain monitor even if the quorum is not formed.

so if it is required to connect to the cluster *before* pinging a monitor, on one side, we are able to reference a monitor by its "name" instead of "id", but on the flip side, we will not be able to ping a monitor until the quorum is formed and the monitors are "readable".

i think it's a trade off. and i think it would be desirable to keep this feature.